playwright/src/test/java/com/microsoft/playwright/TestTap.java

```diff
@@ -103,11 +104,7 @@ void shouldWaitForANavigationCausedByATap() throws InterruptedException {
     List<String> events = Collections.synchronizedList(new ArrayList<>());
     server.setRoute("/intercept-this.html", exchange -> {
       // make sure the tap doesnt resolve too early
-      try {
-        Thread.sleep(100);
-      } catch (InterruptedException e) {
-        events.add("interrupted");
-      }
+      assertDoesNotThrow(() -> Thread.sleep(100));
```

Comment:
based on the fact that the assert at the end of the test does not expect `"interrupted"` to be in the array we are actually expecting this call to not throw